packages/core/core/src/services/document-service/relations/utils/types.ts

```diff
@@ -0,0 +1,5 @@
+// TODO: Shorthand id should only be a string on documents.
+// It's also a number here to make it works with existing V4 types.
+export type ID = string | number;
```

Comment:
At what point do we break everything?

Reply:
Good point.
My proposal is to break it when the api is stable and provides the basic needs for v5. So probably after relations an uniqueness we might be refactoring types

Reply:
Okay, maybe we do that on another branch where the app can be "broken" and we work together on it?